src/modules/roles/routes/getRoleRoute.js

```diff
@@ -0,0 +1,18 @@
+const {Router} = require('express');
+const validate = require('../../user/middlewares/validation');
+const {getRoleSchema} = require('../schema/roleSchema');
+const getRole = require('../controllers/getRole');
+const router = Router();
+
+
+router.post('/', async(req, res, next) => {
```

Comment:
file says 'getRoleRoute' and method is post with body, kindly correct it accordingly